src/toast/toast.styles.tsx

```diff
@@ -0,0 +1,159 @@
+import styled from "styled-components";
+import { Color } from "../color/color";
+import { NotificationType } from "./types";
+import { IconButton } from "../icon-button";
+import { TextStyleHelper } from "../text";
+import { Text } from "../text";
+import { CrossIcon } from "@lifesg/react-icons/cross";
+
+interface StyleProps {
+    $type: NotificationType;
+}
+
+export const Wrapper = styled.div<StyleProps>`
+    width: 100%;
+    padding: 16px;
+    border-radius: 8px;
+
+    ${(props) => {
+        switch (props.$type) {
+            case "success":
+                return `
+                    background: ${Color.Validation.Green.Background(props)};
+                    border: 1pt solid ${Color.Validation.Green.Border(props)};
+                    color: ${Color.Validation.Green.Text(props)};
+                    svg {
+                        font-size: 24px !important;
+                        color: ${Color.Validation.Green.Icon(props)}
+                    }
+                `;
+            case "warning":
+                return `
+                    background: ${Color.Validation.Orange.Background(props)};
+                    border: 1pt solid ${Color.Validation.Orange.Border(props)};
+                    color: ${Color.Validation.Orange.Text(props)};
+                    svg {
+                        font-size: 24px !important;
+                        color: ${Color.Validation.Orange.Icon(props)}
+                    }
+                `;
+            case "error":
+                return `
+                background: ${Color.Validation.Red.Background(props)};
+                border: 1pt solid ${Color.Validation.Red.Border(props)};
+                color: ${Color.Validation.Red.Text(props)};
+                svg {
+                    font-size: 24px !important;
+                    color: ${Color.Validation.Red.Icon(props)}
+                }
+            `;
+            case "info":
+                return `
+                background: ${Color.Validation.Blue.Background(props)};
+                border: 1pt solid ${Color.Validation.Blue.Border(props)};
+                color: ${Color.Validation.Blue.Text(props)};
+                svg {
+                    font-size: 24px !important;
+                    color: ${Color.Validation.Blue.Icon(props)}
+                }
+            `;
+            default:
+                return `
+                    background: ${Color.Validation.Green.Background(props)};
```

Comment:
i see alot of repeats. Possible to use the switch case to determine the set instead. E.g.
```suggestion
let elementAttrs;
switch (props.$type) {
     case "success":
            elementAttrs = Color.Validation.Green;
             break;
    // carry on for others and declare a single return at the end
}